packages/paste-types/src/style-props/backgroundColor.ts

```diff
@@ -2,5 +2,5 @@ import {ThemeShape} from '@twilio-paste/theme-tokens';
 import {ResponsiveValue} from 'styled-system';
 
 export interface BackgroundColorProps {
-  backgroundColor?: ResponsiveValue<keyof ThemeShape['backgroundColors']>;
+  backgroundColor?: ResponsiveValue<keyof ThemeShape['backgroundColors']> | ResponsiveValue<keyof ThemeShape['borderColors']>;
```

Comment:
So we don't want to make this change because this opens a huge can of worms where people can use borderColors for backgroundColors.  These types are going to be used to define everything, so this would break our system.

Reply:
Makes sense, thanks for the update